app/v2/connectors/ConversionConnector.scala

```diff
@@ -70,14 +72,20 @@ class ConversionConnectorImpl @Inject() (httpClientV2: HttpClientV2, appConfig:
           .addHeaders(HeaderNames.CONTENT_TYPE -> MimeTypes.JSON)
           .addHeaders(HeaderNames.ACCEPT -> MimeTypes.XML)
           .withBody(xmlStream)
```

Comment:
can we rename xmlStream to jsonStream?

Reply:
The input is an xmlStream, then after the conversion it will return a jsonStream

Reply:
No, it is actually the other way around!

Reply:
Oh yeah